src/applications/travel-pay/components/TravelClaimCard.jsx

```diff
@@ -2,19 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { format } from 'date-fns';
 
-const CLAIM_STATUS = {
-  SAVED: 'Saved',
```

Comment:
it looks like we still need to translate the statuses from the data - for example, one of the statuses we get is `"InManualReview"`, which should be translated to `"In Manual Review"`

Reply:
never mind - I see that this is done in the backend now